transformations/concept2sentence/transformation.py

```diff
@@ -0,0 +1,70 @@
+import itertools
+import random
+
+from interfaces.SentenceOperation import SentenceAndTargetOperation
+from tasks.TaskTypes import TaskType
+
+from sibyl import Concept2Sentence
+
+
+class C2S(SentenceAndTargetOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION
+    ]
+    languages = ["en"]
```

Comment:
Please add the relevant [keywords](https://github.com/GEM-benchmark/NL-Augmenter/blob/main/docs/keywords.md) too.

Reply:
Done